squid/squid.conf

```diff
@@ -0,0 +1,7133 @@
+#	WELCOME TO SQUID 3.3.8
```

Comment:
Could this file be made ~7,100 lines shorter by removing all the comments excepting those directing folks towards documentation sources?